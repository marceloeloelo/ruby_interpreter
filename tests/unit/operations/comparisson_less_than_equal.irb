true
false
true
true
false
true
true
true
true
false
true
true
true
true
ArgumentError: comparison of String with 1 failed
ArgumentError: comparison of Fixnum with String failed
ArgumentError: comparison of Fixnum with String failed
ArgumentError: comparison of String with Float failed
ArgumentError: comparison of String with Float failed
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for true:TrueClass
ArgumentError: comparison of Fixnum with true failed
ArgumentError: comparison of Fixnum with true failed
ArgumentError: comparison of Fixnum with true failed
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for false:FalseClass
ArgumentError: comparison of Fixnum with false failed
ArgumentError: comparison of Fixnum with false failed
ArgumentError: comparison of Fixnum with false failed
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for nil:NilClass
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for nil:NilClass
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for true:TrueClass
ArgumentError: comparison of String with true failed
ArgumentError: comparison of String with true failed
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for false:FalseClass
ArgumentError: comparison of String with false failed
ArgumentError: comparison of String with false failed
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for true:TrueClass
NoMethodError: undefined method `<=' for true:TrueClass
ArgumentError: comparison of Float with true failed
ArgumentError: comparison of Float with true failed
ArgumentError: comparison of Float with true failed
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for false:FalseClass
NoMethodError: undefined method `<=' for false:FalseClass
ArgumentError: comparison of Float with false failed
ArgumentError: comparison of Float with false failed
ArgumentError: comparison of Float with false failed
NoMethodError: undefined method `<=' for nil:NilClass
ArgumentError: comparison of Fixnum with nil failed
ArgumentError: comparison of Fixnum with nil failed
ArgumentError: comparison of Fixnum with nil failed
NoMethodError: undefined method `<=' for nil:NilClass
NoMethodError: undefined method `<=' for nil:NilClass
NoMethodError: undefined method `<=' for nil:NilClass
ArgumentError: comparison of Float with nil failed
ArgumentError: comparison of Float with nil failed
ArgumentError: comparison of Float with nil failed
NoMethodError: undefined method `<=' for nil:NilClass
NoMethodError: undefined method `<=' for nil:NilClass
NoMethodError: undefined method `<=' for nil:NilClass
