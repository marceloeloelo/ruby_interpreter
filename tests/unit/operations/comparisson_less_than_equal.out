true
false
true
true
false
true
true
true
true
false
true
true
true
true
TypeError: Fixnum can't be coerced into String
